README.md

```diff
@@ -26,13 +26,16 @@ ember install ember-frost-object-browser
 
 ### Action Bar
 
-| Attribute        | Type          | Value | Description                                         |
-| ---------------- | ------------- | ----- | --------------------------------------------------- |
-| controls         | `array`       |       | Controls that will be available in the action bar   |
-| isLoading        | `boolean`     | false | **default** - Action bar is not in a loading state  |
-|                  |               | true  | Action bar is in a loading state                    |
-| loadingText      | `string`      |       | Text that appears beside the loading animation      |
-| selectedItems    | `array`       |       | List of items that are currently selected           |
+| Attribute       | Type                                        | Value | Description                                         |
+| --------------- | ----------------------------------          | ----- | --------------------------------------------------- |
+| controls        | `array`                                     |       | Controls that will be available in the action bar   |
+| isLoading       | `boolean`                                   | false | **default** - Action bar is not in a loading state  |
+|                 |                                             | true  | Action bar is in a loading state                    |
+| forceVisible    | `boolean`                                   |       | forces the component's visibility                   |
```

Comment:
I don't understand how this fits into this PR.  I didn't see anything mentioned about it in the changelog and it does not seem related to the new behavior.  Is it left over from something else or was it intended to be added as well?  Would `isVisible` align more with `isLoading`?  I'm not sure I'm sold on the `force` part.

Reply:
So, by default this component only shows when selected items has length. I looked at a few of our apps, and there are places where the functionality I'm adding will be wanted with the ability to force the action bar component to be visible all the time. I should probably add something about it to the changelog. Perhaps `alwaysVisible` would be better?

Reply:
I like `alwaysVisible`.  And do please add an entry in the changelog